### PR TITLE
flickering

### PR DESCRIPTION
**Pull request by waynepHMCTS**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
* Stop search location no result optons from showing before result off search.
*
*

### CODE REVIEW

File: projects/exui-common-lib/src/lib/components/search-location/search-location.component.spec.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { of } from 'rxjs';
 import { LocationService } from '../../services/locations/location.service';
 import { SearchLocationComponent } from './search-location.component';
 
-describe('SearchLocationComponent', () => {
+fdescribe('SearchLocationComponent', () => {
```
Comment (Kartheek-S) [inline projects/exui-common-lib/src/lib/components/search-location/search-location.component.spec.ts:10]: fdescribe

Comment (waynepHMCTS) [reply projects/exui-common-lib/src/lib/components/search-location/search-location.component.spec.ts:10]: arrrrgh again

**Review by Kartheek-S — APPROVED** [review]

File: projects/exui-common-lib/src/lib/components/search-location/search-location.component.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
       [matAutocomplete]="autoSearchLocation"
       (keydown)="onKeyDown()"
       (keyup)="onKeyUp($event)"
+      (focus)="onFocus()"
```
Comment (rayliangatos) [inline projects/exui-common-lib/src/lib/components/search-location/search-location.component.html:8]: Is this the only solution to fix this issue? As I'm seeing three internal events are used here, keydown/keyup/focus, when keydown/keyup is triggered focus is probably triggered as well at the same time, any side effect to bind all the events together? I'm a bit concerned on this but if all tests are fine that will be okay.

Comment (waynepHMCTS) [reply projects/exui-common-lib/src/lib/components/search-location/search-location.component.html:8]: All tests are fine

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved

**Pull request merged**